### DISCUSSION TITLE
Inconsistent output from API

### DISCUSSION THREAD

**Discussion by paulchoong** [perplexityai/api-discussion#38] [Q&A] [answered]
Using the API with default settings, I've used this prompt below. I'm getting inconsistent output. Most of the time it's markdown with occasions a combination of markdown and html. In one rare occasion, I got it exactly how I wanted it to be which is in html. How is it possible to force HTML output only? I've tried asking response to be HTML only and it still don't work. Thank you very much for your help

`You are the chief news editor. Do a full research on the topic and write a compilation article including all information from sources. Write in 3rd person making sure to mention names and quotes as they are. Be sure not make them up. Do not include date and location at the start of your output. Display list of sources at the end of body but do not repeat them. Write in UK English. Do not add a title. Your response must be in HTML code and not Markdown.`

**Comment by dfsklar** [comment]
Here we are in 2025 and still -- I get the same hybrid mix of markdown and HTML (if I'm lucky) and never any consistency.  How can this be an impossible ask?

**Reply by paulchoong** [reply, marked as answer]
The solution is just to use ChatGPT 😀

**Comment by satounity** [comment]
Just output it in markdown and convert it to html using a library.

**Reply by paulchoong** [reply]
As I mentioned, the output is inconsistent. You can't get all markdown or all html. Try it enough times, you can get a mixture of both or neither
